### DISCUSSION TITLE
Core details edit using API? When?

### DISCUSSION THREAD

**Discussion by SrdjanPaunovic** [etsy/open-api#1291] [Q&A]
Hello everyone,

I have a question regarding the ability to edit the core details of a listing using the Etsy API v3. In the Etsy UI, users can now select additional options in the core details, but it seems we are unable to do the same via the API.

Is there a possibility that this feature is available but not yet reflected in the API documentation? Any insights or updates on this would be appreciated.

Thank you!

![image](https://github.com/user-attachments/assets/5de661cb-6af0-485c-8001-1f361875e835)

**Comment by cyrilchandelier** [comment]
AFAIK it's not available yet. I emailed developer@etsy.com to ask about it and will report here if I hear back, but you should maybe also email them as more inbound may help them prioritize.

**Reply by cyrilchandelier** [reply]
Oh well, they responded saying:
> We currently do not have any plans to add these fields to our API. But, I have noted your request and shared it with my team. We rely on feedback from our members as we continue to make improvements in the best interest of the Etsy Community.
>  
> We don't have a definite time frame on when the process will be complete, nor are we able to share out the long term focus or goals of this process. But once we have more to share, we will let everybody know via our GitHub repo.
> 
> To stay apprised of upcoming changes in Etsy's API, please see this GitHub post: https://github.com/etsy/open-api/discussions/1207.

**Reply by SrdjanPaunovic** [reply]
We still haven't gotten any response here...

**Comment by showtroylove** [comment]
At first glance, this appears to describe the listing properties, but I need more information about the taxonomy of the listing to assist further. Specifically, could you provide more details about the listing, such as the taxonomy ID? This ID is critical as it helps categorize your product accurately on Etsy’s platform, ensuring it is placed in the appropriate section, like "Jewelry > Necklaces > Chokers".

**Reply by SrdjanPaunovic** [reply]
How is this relevant to the topic?

**Comment by SrdjanPaunovic** [comment]
@nateswart @pnaf can you guys tell us more about this? I still didn't get any response from [developer@etsy.com](mailto:developer@etsy.com)

**Comment by showtroylove** [comment]
I have provided many responses to this and other similar questions. Perform
a search to find it.

**Reply by SrdjanPaunovic** [reply]
I didn't have luck finding it. Could you share links here? 
Similar topic with no response:
https://github.com/etsy/open-api/discussions/1269
https://github.com/etsy/open-api/discussions/1340

**Comment by showtroylove** [comment]
It seems your question doesn't provide enough information, so my responses
are of a general nature. Etsy will likely place that new property in its
listing properties repository:
https://developers.etsy.com/documentation/reference/#operation/getListingProperties
.

For intellectual property purposes, I wouldn't address that question, and
it's not necessary. If you say digital content and you're US-based, Etsy
will likely ignore any claim that the AI-generated images were duplicated
or stolen. Why? US case law has declared that AI-generated images/content
are not copyright-protected, and therefore, anyone can copy and resell the
content. This is NOT legal advice, but a path you should discuss with your
attorney.

**Comment by showtroylove** [comment]
See my other response and mark the response as answered if appropriate.
The help provided by forum members is voluntary and it gives credit and
closure to the post.

**Comment by SrdjanPaunovic** [comment]
Sorry @showtroylove, but it seems the response doesn't address the question I asked. Allow me to clarify:

Question:
How can we edit the core details of a listing using API v3 ('How is this digital content created?' part)?

Currently, my users are forced to manually change every listing through the Etsy UI (see the image below). This is a significant inefficiency. We need fields in the [updateListing](https://developers.etsy.com/documentation/reference/#operation/updateListing) and [createDraftListing](https://developers.etsy.com/documentation/reference/#operation/createDraftListing) endpoints to enable this functionality programmatically.

It's surprising that this capability isn't included yet, as it's critical for streamlining the workflow. Could you provide a solution or clarify if this feature is on the roadmap?

![image](https://github.com/user-attachments/assets/a97fc096-7be9-4273-8b7a-2522cf07cca4)

**Comment by showtroylove** [comment]
I have already answered this question in detail and generally. You should
look deeper at what I have already supplied. I don't work for Etsy and have
my own users. It is not incumbent upon me to provide solutions. I believe I
have provided information with all that you need to accomplish 99% of the
properties on a listing (new or otherwise).

**Reply by cyrilchandelier** [reply]
Let's stop this discussion here, please, as it's not going anywhere. @showtroylove is not affiliated with Etsy and does not know the answer to @SrdjanPaunovic's question, which was perfectly clear in the first place. We are all suffering from not receiving straight/fast answers from Etsy, we don't need [sassy comments](https://github.com/etsy/open-api/discussions/1291#discussioncomment-11653186) from community members on top of that.

The closest we have to an answer is what Etsy responded in an email:
> We currently do not have any plans to add these fields to our API.

**Reply by SrdjanPaunovic** [reply]
Thank you @cyrilchandelier , I agree.
Hope someone from Etsy will notice and update us about it soon.

They are pretty slow on email responses too. I didn't get any response on this topic..

**Comment by showtroylove** [comment]
I understand that your remarks may have been intended in jest, but they
are not helpful. The question might not be as clear as it seems. The image
provided appears vague and non-descriptive. If this is the response to
someone who takes the time to assist you, it can be discouraging.

**Comment by showtroylove** [comment]
As stated many times now, it appears that your question may not have
provided enough information for a precise answer. I have shared the
solution, and it seems Etsy's tier one support may not be aware of the
detailed workings of the API since they aren't the actual coders or API
designers.

Review the information I provided. All attributes for the listings are
included, except, perhaps, the AI-specific one, but I doubt it, for the new
ones you mentioned. Without reinventing the wheel, it is likely that these
new attributes would be CRUD in the same or a similar manner.

Once again, I encourage you to examine this thoroughly. This is my final
reply on this matter. Alternatively, you can continue waiting for a reply
from Etsy.

**Reply by SrdjanPaunovic** [reply]
I didn’t mean to be rude, but my question was perfectly clear from the start, and I even reformulated it later to make it as clear as possible. Yet, you continue referring to updating properties, which are unrelated to the issue at hand.
I appreciate your input, but let wait to see if someone from Etsy say something more. I sent one more email...

**Comment by SrdjanPaunovic** [comment]
Latest update from Etsy via email (nothing new):

> At this time, updating listing core details via our API is not available, specifically the "How is this digital content created?" section.
>  
> We currently do not have any plans to add these fields to our API but, I have noted your request and shared it with my team. We rely on feedback from our members as we continue to make improvements in the best interest of the Etsy Community.
>  
> We don't have a definite time frame on when the process will be complete, nor are we able to share out the long term focus or goals of this process. But once we have more to share, we will let everybody know via our GitHub repo.

**Comment by ramibch** [comment]
I managed to update just 3 listing fields (title, description and tags). Trying to update other fields was not possible for me using the API v3.
